src/ApparentHorizons/StrahlkorperDataBox.hpp

```diff
@@ -0,0 +1,208 @@
+template <typename Frame>
+struct InvHessian : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "InvHessian";
+  static typename types<Frame>::InvHessian function(
+      const typename types<Frame>::ThetaPhi& theta_phi) noexcept;
+  using argument_tags = typelist<ThetaPhi<Frame>>;
+};
+
+/// (Euclidean) distance \f$r(\theta,\phi)\f$ from the center to each
+/// point of the surface.
+template <typename Frame>
+struct Radius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "Radius";
+  SPECTRE_ALWAYS_INLINE static auto function(
+      const ::Strahlkorper<Frame>& strahlkorper) noexcept {
+    return strahlkorper.ylm_spherepack().spec_to_phys(
+        strahlkorper.coefficients());
+  }
+  using argument_tags = typelist<Strahlkorper<Frame>>;
+};
+
+/// \f$(x,y,z)\f$ of each point on the surface.
+template <typename Frame>
+struct CartesianCoords : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "CartesianCoords";
+  static typename types<Frame>::Vector function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::OneForm& r_hat) noexcept;
+  using argument_tags =
+      typelist<Strahlkorper<Frame>, Radius<Frame>, Rhat<Frame>>;
+};
+
+/// dx_radius(i) is \f$\partial r/\partial x^i\f$.
+/// Here \f$r=r(\theta,\phi)\f$ is the surface, which is considered a
+/// function of Cartesian coordinates \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$
+/// for this operation.
+template <typename Frame>
+struct DxRadius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "DxRadius";
+  static typename types<Frame>::OneForm function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::InvJacobian& inv_jac) noexcept;
+  using argument_tags =
+      typelist<Strahlkorper<Frame>, Radius<Frame>, InvJacobian<Frame>>;
+};
+
+/// d2x_radius(i,j) is \f$\partial^2 r/\partial x^i\partial x^j\f$.
+/// Here \f$r=r(\theta,\phi)\f$ is the surface, which is considered a
+/// function of Cartesian coordinates \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$
+/// for this operation.
+template <typename Frame>
+struct D2xRadius : db::ComputeItemTag {
+  static constexpr db::DataBoxString label = "D2xRadius";
+  static typename types<Frame>::SecondDeriv function(
+      const ::Strahlkorper<Frame>& strahlkorper, const DataVector& radius,
+      const typename types<Frame>::InvJacobian& inv_jac,
+      const typename types<Frame>::InvHessian& inv_hess) noexcept;
+  using argument_tags = typelist<Strahlkorper<Frame>, Radius<Frame>,
+                                 InvJacobian<Frame>, InvHessian<Frame>>;
+};
+
+/// \f$\nabla^2 radius\f$, the flat Laplacian of the surface.
+/// This is \f$\eta^{ij}\partial^2 r/\partial x^i\partial x^j\f$,
+/// where \f$r=r(\theta(x,y,z),\phi(x,y,z))\f$.
+template <typename Frame>
+struct NablaSquaredRadius : db::ComputeItemTag {
```

Comment:
Not a request, but an alternative name that doesn't refer to latex commands would be `LapacianRadius`